virtwho/virt/fakevirt/fakevirt.py

```diff
@@ -1,10 +1,39 @@
 
 import json
 
+from virtwho.config import VirtConfigSection, accessible_file
 from virtwho.virt import Virt, VirtError, Guest, Hypervisor
 from virtwho.util import decode
 
 
+class FakeVirtConfigSection(VirtConfigSection):
+    VIRT_TYPE = 'fake'
+
+    def __init__(self, *args, **kwargs):
+        super(FakeVirtConfigSection, self).__init__(*args, **kwargs)
+
+        self.add_key('file', validation_method=self._validate_fake_virt_file, required=True)
```

Comment:
Just question. What about other virt. backends? They do not have any required options? We solve it in different way now, but `required=True` at some places could simplify code a little. I guess e.g. ESX has to have some required options: `server`, etc.

Reply:
Already in progress in fix_config_refactor_tests_2.